language-design.md

```diff
@@ -106,7 +282,29 @@ you do so? If not, why not?
 
 **Response**
 
-
+No, we don't believe that the two experiments disagree with each other. Pavlus 
+is not arguing for a "natural language" but more for a language that is easier 
+to understand by using words from our current language. This is done by changing 
+operators and key words from symbols into words or symbols that are common to 
+our everyday language, like changing "for(int i = 0; i < 10; i++)" to 
+"integer i = 0 \break repeat 10 times \break i = i + 1" [Pavlus, 2012]. Both 
+examples have the exact same functionality, but change the assumed knowledge of 
```

Comment:
in the change you suggest where would you put the meat of your loop? assuming the meat is not just incrementing i